src/services/arweave/AntContract.ts

```diff
@@ -38,6 +51,34 @@ export class ANTContract {
   set ticker(ticker: string) {
     this.contract.ticker = ticker;
   }
+  get records() {
+    return this.contract.records;
+  }
+  set records(records: { [x: string]: ANTContractDomainRecord }) {
+    //records example: {'@': {transactionId:"7waR8v4STuwPnTck1zFVkQqJh5K9q9Zik4Y5-5dV7nk"}} - can leave the other properties undefined, will autoset
```

Comment:
Something to do here? If so let's update the comment (put ref to some ticket maybe), if not we can remove this line :)

Reply:
No this was just an explainer / me figuring out what the function would do as i was writing it, but the code is obvious enough i guess. Removed.